CMakeLists.txt

```
@@ -625,16 +660,21 @@ target_link_libraries(v8_snapshot
     v8-bytecodes-builtin-list
 )
 
+if (is-riscv)
+	set(mksnp-prefix ${RISCV_EMULATOR} -P --)
```

```suggestion
  set(mksnp-prefix ${RISCV_EMULATOR} -P --)
```
And consider naming it mksnapshot-prefix for clarity.